src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> lines = readFromFile(fromFileName);
```

Comment:
this method is already incorrect - it does too many things

you should have separate methods for reading data, calculating report and writing date